app/javascript/widget/components/Form/PhoneInput.vue

```diff
@@ -157,8 +136,8 @@ function scrollToFocusedOrActiveItem(item) {
   if (focusedOrActiveItemLocal.length > 0) {
     const dropdown = dropdownRef.value;
     const dropdownHeight = dropdown.clientHeight;
-    const itemTop = focusedOrActiveItem[0].offsetTop;
-    const itemHeight = focusedOrActiveItem[0].offsetHeight;
+    const itemTop = focusedOrActiveItemLocal[0]?.offsetTop;
+    const itemHeight = focusedOrActiveItemLocal[0]?.offsetHeight;
```

Comment:
These changes will fix the console errors.

<img width="829" alt="image" src="https://github.com/user-attachments/assets/8a00502f-a04b-4090-98a2-e3424d984b72">